packages/components/src/components/hds/tooltip-button/index.ts

```diff
@@ -16,7 +16,7 @@ export interface HdsTooltipSignature {
     extraTippyOptions: Omit<TippyProps, 'placement' | 'offset'>;
```

Comment:
`@extraTippyOptions` should be optional as well

```suggestion
    extraTippyOptions?: Omit<TippyProps, 'placement' | 'offset'>;
```